### PR TITLE
Changing max-line-length to 115 for flake8

### PR DESCRIPTION
**Pull request by ollie-iterators**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |
| ✓   | :sparkles: New feature |
| ✓   | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Changing max-line-length to 115 for flake8. This is so that the documentation is easier to read and so that there are not too many changes to make at a time for any file.

NOTE: I know that there is an error in some files, but I am worried that at least one of them are impacting tests so I think it would be better if someone else moved it.

Refs #XXXX

Closes #XXXX

### CODE REVIEW

File: pylint/extensions/typing.py [hunk 1]
```diff
@@ -127,7 +127,8 @@ class TypingChecker(BaseChecker):
         "R6006": (
             "Type `%s` is used more than once in union type annotation. Remove redundant typehints.",
             "redundant-typehint-argument",
-            "Duplicated type arguments will be skipped by `mypy` tool, therefore should be removed to avoid confusion.",
+            "Duplicated type arguments will be skipped by `mypy` tool, therefore should be removed"
```
Comment (Pierre-Sassoulas) [inline pylint/extensions/typing.py:130]:
```suggestion
            "Duplicated type arguments will be skipped by `mypy`, therefore should be removed"
```

File: pylint/reporters/text.py [hunk 2]
```diff
@@ -132,8 +132,13 @@ def colorize_ansi(
     """
     # TODO: 3.0: Remove deprecated typing and only accept MessageStyle as parameter
     if not isinstance(msg_style, MessageStyle):
+        WARNING_MESSAGE = (
+            "In pylint 3.0, the colorize_ansi function of Text reporters will only "
+        )
+        WARNING_MESSAGE += "accept a MessageStyle parameter"
```
Comment (Pierre-Sassoulas) [inline pylint/reporters/text.py:138]: Please don't use additional assignments, you can use parenthesis and implicit string concatenation I've fixed/suggested that three time already :smile:

File: pylint/testutils/_primer/primer_compare_command.py [hunk 3]
```diff
@@ -128,7 +128,8 @@ def _create_comment_for_package(
             assert not self.packages[package].url.endswith(
                 ".git"
             ), "You don't need the .git at the end of the github url."
-            comment += f"{self.packages[package].url}/blob/{new_messages['commit']}/{filepath}#L{message['line']}\n"
+            comment += f"{self.packages[package].url}/blob/{new_messages['commit']}"
+            comment += f"/{filepath}#L{message['line']}\n"
```
Comment (Pierre-Sassoulas) [inline pylint/testutils/_primer/primer_compare_command.py:132]: Ditto

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Great !

File: pylint/reporters/text.py (hunk 2 above)
Comment (ollie-iterators) [reply pylint/reporters/text.py:138]: Oops, sorry.

File: pylint/extensions/typing.py (hunk 1 above)
Comment (ollie-iterators) [reply pylint/extensions/typing.py:130]: Fixed

**Comment by ollie-iterators** [comment]
Closing until fixed

**Comment by Pierre-Sassoulas** [comment]
Don't worry you don't have to close :) You should fix locally before pushing to save runner time though.

**Pull request closed without merging**